lib/js_routes.rb

```diff
@@ -36,6 +36,7 @@ class JsRoutes
 
   LAST_OPTIONS_KEY = "options".freeze
   FILTERED_DEFAULT_PARTS = [:controller, :action, :subdomain]
+  URL_OPTIONS = [:protocol, :domain, :host, :port]
```

Comment:
It could be there, but `subdomain` is also in `FILTERED_DEFAULT_PARTS`, so the check will be 
```ruby
FILTERED_DEFAULT_PARTS.exclude?(:subdomain) && URL_OPTIONS.include?(:subdomain)
```
and this is always false. Of course, I can add it.